### PR TITLE
[5.1] Added afterLoadingEnvironment() test

### PR DESCRIPTION
**Pull request by bancur**
Add test for afterLoadingEnvironment method.

### CODE REVIEW

File: tests/Foundation/FoundationApplicationTest.php [hunk 1]
```diff
@@ -123,6 +123,17 @@ public function testEnvironment()
...
         $this->assertFalse($app->environment(['qux', 'bar']));
     }
+   
+    public function testMethodAfterLoadingEnvironmentAddsClosure()
+    {
+        $app = new Application;
+        $closure = function(){};
+        $app->afterLoadingEnvironment($closure);
+        $eventName = 'bootstrapped: '
+                    .'Illuminate\Foundation\Bootstrap\DetectEnvironment';
```
Comment (GrahamCampbell) [inline tests/Foundation/FoundationApplicationTest.php]: please keep this on one line

File: tests/Foundation/FoundationApplicationTest.php [hunk 2]
```diff
@@ -123,6 +123,17 @@ public function testEnvironment()
         $this->assertFalse($app->environment('qux', 'bar'));
         $this->assertFalse($app->environment(['qux', 'bar']));
     }
+   
+    public function testMethodAfterLoadingEnvironmentAddsClosure()
+    {
+        $app = new Application;
+        $closure = function(){};
```
Comment (GrahamCampbell) [inline tests/Foundation/FoundationApplicationTest.php]: cs

Comment (GrahamCampbell) [inline tests/Foundation/FoundationApplicationTest.php]: also, the variable asignment is unneeded

**Comment by bancur** [comment]
Can you check if it's OK now?

File: tests/Foundation/FoundationApplicationTest.php [hunk 3]
```diff
@@ -123,6 +123,15 @@ public function testEnvironment()
         $this->assertFalse($app->environment('qux', 'bar'));
         $this->assertFalse($app->environment(['qux', 'bar']));
     }
+
+    public function testMethodAfterLoadingEnvironmentAddsClosure()
+    {
+        $app = new Application;
+        $closure = function () {};
+        $app->afterLoadingEnvironment($closure);
+        $this->assertTrue(isset($app['events']->getListeners('bootstrapped: Illuminate\Foundation\Bootstrap\DetectEnvironment')[0]));
```
Comment (GrahamCampbell) [inline tests/Foundation/FoundationApplicationTest.php]: use assertArrayHasKey

**Comment by bancur** [comment]
Changed the assertion to assertArrayHasKey.

**Pull request merged**